README.rst

```diff
@@ -35,9 +35,16 @@
 .. |Tracking| image:: https://img.shields.io/badge/issue_tracking-github-blue
         :target: https://github.com/diffpy/diffpy.nmf_mapping/issues
 
-Run NMF analysis on PDF and XRD data
+Python package for running NMF analysis on PDF and XRD data.
```

Comment:
I copied the longer description but I think we can refine it. Will address in a separate PR.